test/main.test.js

```diff
@@ -433,14 +433,18 @@ describe('downloading feeds', () => {
         expect(body).toMatchSnapshot();
     });
 
-    test('/feed/:file', () => get(`/feed/doesnotexist`).expect(400));
```

Comment:
this test was asserting something which the name did not indicate. added separate tests for the different cases.

Reply:
(that is, we have joi validation that the file is named `something.{css,json,js}`)